dspace-api/src/main/java/org/dspace/app/canvasdimension/CanvasDimensionCLI.java

```diff
@@ -0,0 +1,202 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.canvasdimension;
+
+import java.util.Arrays;
+import java.util.UUID;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.MissingArgumentException;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.dspace.app.canvasdimension.factory.IIIFCanvasDimensionServiceFactory;
+import org.dspace.app.canvasdimension.service.IIIFCanvasDimensionService;
+import org.dspace.content.Collection;
+import org.dspace.content.Community;
+import org.dspace.content.DSpaceObject;
+import org.dspace.content.Item;
+import org.dspace.core.Constants;
+import org.dspace.core.Context;
+import org.dspace.eperson.EPerson;
+import org.dspace.eperson.factory.EPersonServiceFactory;
+import org.dspace.eperson.service.EPersonService;
+import org.dspace.handle.factory.HandleServiceFactory;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
+
+/**
+ * Sets IIIF canvas metadata on bitstreams based on image size.
+ *
+ * @author Michael Spalti mspalti@willamette.edu
+ */
+public class CanvasDimensionCLI {
```

Comment:
I agree that this should eventually be updated. Perhaps an issue for 7.3

Reply:
CurationTask is also something to consider as it can be run as  part of the workflow. We have it in our addon so it is something that we can eventually provide that for 7.3+